zerver/openapi/zulip.yaml

```diff
@@ -5023,6 +5023,201 @@ paths:
+                              "field_data": "",
+                              "order": 3,
+                            },
+                            {
+                              "id": 4,
+                              "name": "Favorite editor",
+                              "type": 3,
+                              "hint": "",
+                              "field_data": '{"vim":{"text":"Vim","order":"1"},"emacs":{"text":"Emacs","order":"2"}}',
+                              "order": 4,
+                            },
+                            {
+                              "id": 5,
+                              "name": "Birthday",
+                              "type": 4,
+                              "hint": "",
+                              "field_data": "",
+                              "order": 5,
+                            },
+                            {
+                              "id": 6,
+                              "name": "Favorite website",
+                              "type": 5,
+                              "hint": "Or your personal blog's URL",
+                              "field_data": "",
+                              "order": 6,
+                            },
+                            {
+                              "id": 7,
+                              "name": "Mentor",
+                              "type": 6,
+                              "hint": "",
+                              "field_data": "",
+                              "order": 7,
+                            },
+                            {
+                              "id": 8,
+                              "name": "GitHub",
+                              "type": 7,
+                              "hint": "Enter your GitHub username",
+                              "field_data": '{"subtype":"github"}',
+                              "order": 8,
+                            },
+                          ],
+                      }
+    patch:
+      operationId: reorder_custom_profile_fields
+      tags: ["server_and_organizations"]
+      description: |
+        {!api-admin-only.md!}
+
+        Reorder the custom profile fields in the user's organization.
+
+        `PATCH {{ api_url }}/v1/realm/profile_fields`
+      parameters:
+        - name: order
+          in: query
+          description: |
+            A list of ids of the respective custom profile fields in the desired
+            order.
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  type: integer
+              example: [10, 9, 8, 7, 6, 5, 4, 3, 2, 1]
+          required: true
+      responses:
+        "200":
+          description: Success.
+          content:
+            application/json:
+              schema:
+                allOf:
+                  - $ref: "#/components/schemas/JsonSuccess"
```

Comment:
```
    JsonSuccess:
      allOf:
        - $ref: "#/components/schemas/JsonResponse"
        - required:
            - result
            - msg
        - properties:
            result:
              enum:
                - success
            msg:
              type: string
        - example: {"msg": "", "result": "success"}
```
JsonSuccess already has an example so I don't think there's a need to add example.

Reply:
Sorry for nitpicking but you forgot to remove the `allOf` when removing examples.

Reply:
Oops ! Thanks for pointing that out.